Playmate/View Controllers/Session Details View/PlayerProfileViewController.m

```diff
@@ -46,12 +46,8 @@ - (void)viewDidLoad {
     self.genderLabel.text = [@"Identifies as " stringByAppendingString:self.user[@"gender"][0]];
     self.ageLabel.text = [[Constants getAgeInYears:self.user[@"birthday"][0]] stringByAppendingString:@" years old"];
     
-    if ([self.user objectForKey:@"biography"] != nil) {
-        self.bioField.text = self.user[@"biography"][0];
-    }
-    else {
-        self.bioField.text = @"No biography";
-    }
+    const BOOL hasBiography = ([self.user objectForKey:@"biography"] != nil);
+    self.bioField.text = hasBiography ? self.user[@"biography"][0] : @"No biography";
```

Comment:
I see this repeated, we could consider having a helper to extract data from `self.user`